ldk/node/src/hostClients/whisperService.ts

```diff
@@ -108,6 +108,88 @@ export interface WhisperFormConfig extends Whisper {
   inputs: { [name: string]: WhisperFormInputs };
 }
 
+export interface WhisperListElement<T extends string> {
+  type: T;
+  /**
+   *  Value the UI uses to determine if the element is shown
+   *  in the condensed whisper view. Extra being false means
+   *  the element will always be shown. Extra being true means
+   *  the element will only show in the expanded view.
+   *  If this value is ommited it will default to false.
+   */
+  extra?: boolean;
+  /**
+   *  Value the UI uses to order the form inputs.
+   *  Value must be greater than 0
+   *  If this value is ommited it will default to 0
+   */
+  order?: number;
+}
+
+// eslint-disable-next-line no-shadow
```

Comment:
- [ ] @richseviora You asked me to make a note to remind you to look at this shadow issue and the one for WhisperListAlign as well.